service/awsconfig/v10/adapter/error.go

```diff
@@ -23,23 +23,30 @@ func IsMalformedAmazonAccountID(err error) bool {
 	return microerror.Cause(err) == malformedAmazonAccountIDError
 }
 
+var notFoundError = microerror.New("not found")
+
+// IsNotFound asserts notFoundError
+func IsNotFound(err error) bool {
+	return microerror.Cause(err) == notFoundError
+}
+
```

Comment:
The diff is a pain here because I ordered the errors. Not found is the only new error.